src/registry/partners/edit.tsx

```diff
@@ -0,0 +1,104 @@
+import {PartnersComponentSchema} from '@open-formulieren/types';
+import {useIntl} from 'react-intl';
+
+import {
+  BuilderTabs,
+  ClearOnHide,
+  Description,
+  Hidden,
+  IsSensitiveData,
+  Key,
+  Label,
+  Registration,
+  SimpleConditional,
+  Tooltip,
+  Translations,
+  useDeriveComponentKey,
+} from '@/components/builder';
+import {LABELS} from '@/components/builder/messages';
+import {TabList, TabPanel, Tabs} from '@/components/formio';
+import {useErrorChecker} from '@/utils/errors';
+
+import {EditFormDefinition} from '../types';
+
+/**
+ * Form to configure a Formio 'partners' type component.
+ */
+const EditForm: EditFormDefinition<PartnersComponentSchema> = () => {
+  const intl = useIntl();
+  const [isKeyManuallySetRef, generatedKey] = useDeriveComponentKey();
+  const {hasAnyError} = useErrorChecker<PartnersComponentSchema>();
+
+  return (
+    <Tabs>
+      <TabList>
+        <BuilderTabs.Basic
+          hasErrors={hasAnyError(
+            'label',
+            'key',
+            'description',
+            'tooltip',
+            'hidden',
+            'clearOnHide',
+            'isSensitiveData',
+            'defaultValue'
```

Comment:
this is not a field that can be edited, so tracking validation errors for it makes little sense